luaui/Widgets/dbg_test_runner.lua

```diff
@@ -221,9 +177,126 @@ local testRunState
 local activeTestState
 local resumeState
 local returnState
-local callinState
+local callinState = {callins = {}, recording = {}, unsafe = false}
 local spyControls
 
+
+-- callin tracking
+-- =========
+
+-- Hook callin
+-- Will register to count either predicate success or execution counts.
+-- predicate will be passed the callin arguments.
+-- @string name Callin name
+-- @func predicate Test function or false to just count callin calls
+-- @param target Object registering the callin, default: test_runner widget
+-- @number depth stack depth (normally for internal use)
+function registerCallin(name, predicate, target, depth)
+	local REGISTER_COUNT = 0
+	local REGISTER_FULL = 1
+
+	local depth = depth + 1
```

Comment:
I don't see `depth` being used anywhere.

Reply:
it's used a few lines below at the error calls